lsp-mode.el

```diff
@@ -422,6 +422,49 @@ must be used for handling a particular message.")
   "Face used for highlighting symbols being written to."
   :group 'lsp-faces)
 
+(defcustom lsp-lenses-check-interval 0.1
```

Comment:
Singular form may be good enough... `codeLens` is in its singular form.

Reply:
Make sense, will change it everywhere.